packages/client/hmi-client/src/views/ModelView.vue

```diff
@@ -55,16 +60,16 @@ watch(newModelId, (id) => {
 
 onMounted(async () => {
 	// If no model id provided in props, do not attempt to make inital tab
-	if (props.modelId !== '') {
+	if (props.assetId !== '') {
 		const initialTab = {
-			name: getModelName(props.modelId),
+			name: getModelName(props.assetId),
 			props: {
-				modelId: props.modelId
+				assetId: props.assetId
 			}
 		} as Tab;
 
 		openTabs.value.push(initialTab);
-	} // end if
+	} // end if:show-recents-props="{ project: props.project, resourceRoute: RouteName.ModelRoute }"
```

Comment:
I'm a little confused by this comment change, maybe a multiple-cursor typo?

Reply:
Forgot to get rid of that